hack/internal/pkg/utils/apiutils.go

```diff
@@ -0,0 +1,124 @@
+package utils
```

Comment:
thought (non-blocking): Many of the functions and concepts found within this `apiutils.go` file can also be found in [pkg/controller/build/utils/lookups.go](https://github.com/openshift/machine-config-operator/blob/master/pkg/controller/build/utils/lookups.go). Adopting them here is a bit more involved since the lookup funcs in that library use listers while these use full API clients.

To be clear: I'm just pointing out that duplication exists. I'm not expecting you to dedupe them :smile: